### ISSUE TITLE
Data integrity test for party coverage

### ISSUE THREAD

**Issue by MansMeg** [welfare-state-analytics/riksdagen-corpus#311]
We should check that we have good coverage of party affiliations in the MP database so we can garantee say 95% or 99% party coverage of MPs.

**Comment by JoeNoonan** [comment]
I have pulled out the party lists from the matriklar. This may be of help for quality control as it gives an 'official name' for each of the parties along with an abbreviation. 

One area of possible confusion that in 1933 where there was a split in the communist party with “Sillénarna” and “Kilbommarna”, the latter renamed themselves “Socialistiska partiet” in 1934. It looks like in the metadata the Kilbommarna are called "Kilbomspartiet". In the matriklar it is Sillénarna whose party is called Kominternpartiet which is separate from the Kommunistiska Gruppen. 

[party_list_from_matrikel.csv](https://github.com/welfare-state-analytics/riksdagen-corpus/files/11651632/party_list_from_matrikel.csv)

One other thing is that I have noticed that there is inconsistencies in whether the contemporary name of the party is used, or the name of the party at the time. Take for example Q6147346, he was in office from 1929 to 1939. His value in the party table is "Liberalerna". The party table does not have date values. In contrast, Q6026474, who does have date values, has a party value of "Folkpartiet". Both Liberalerna and Folkpartiet have seperate party_id (Folkpartiet = Q53764745 and Liberalerna = Q110857). Maybe this could be checked with a unit test looking at the unique number of parties in the riksdag per given year?
